### PR TITLE
add apf metrics

### PR DESCRIPTION
**Pull request by tkashem**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description

Add some commonly used API Priority & Fairness metrics

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by jtaleric — COMMENTED** [review]
Thanks @tkashem! It might also be good for us to capture some of these queries here https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/metrics-report.yml (this is in an effort to reduce the number of metrics we collect).

**Comment by tkashem** [comment]
> Thanks @tkashem! It might also be good for us to capture some of these queries here https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/metrics-report.yml (this is in an effort to reduce the number of metrics we collect).

Sure, the metrics have been added there, i left `instant: true` for all of them.

File: cmd/config/metrics-report.yml [hunk 1]
```diff
@@ -8,6 +8,30 @@
   metricName: avg-mutating-apicalls-latency
   instant: true
 
+- query: sum(rate(apiserver_flowcontrol_dispatched_requests_total{}[5m])) by(priority_level)
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply cmd/config/metrics-report.yml:11]: For example, the picture below represents the average rate of the dispatched requests seen in the latest 10 minutes.

image

[the remark this answers is not in the document]
Comment (tkashem) [reply cmd/config/metrics-report.yml:11]: i thought it's going to be translated into a recording rule

**Comment by jtaleric** [comment]
Checking in on this @tkashem / @rsevilla87

**Comment by rsevilla87** [comment]
I can take over this one and add the missing metrics

**Pull request closed without merging**